client/src/components/Bio/Bio.js

```diff
@@ -3,178 +3,171 @@ import "./Bio.css";
+import TextField from "@material-ui/core/TextField";
+import Button from "../../components/Button/Button";
 
-import axios from 'axios';
-import { BASE_URL } from '../../services/constants';
+import axios from "axios";
+import { BASE_URL } from "../../services/constants";
 
 // const Bio = props => {
 class Bio extends Component {
-  constructor(){
+  constructor() {
     super();
     this.state = {
-        editMode: false,
-          form_first_name: '',
-          form_last_name: '',
-          form_bio: ''
-    }
+      editMode: false,
+      form_first_name: "",
+      form_last_name: "",
+      form_bio: ""
+    };
   }
 
   toggleEditMode = () => {
-      this.setState({ 
-        editMode: !(this.state.editMode)
-      })
-    
-  }
+    this.setState({
+      editMode: !this.state.editMode
+    });
+  };
 
-  updateFirstName = (event) => {
+  updateFirstName = event => {
     let first_name = event.target.value;
-    this.setState({ 
-           form_first_name: first_name 
+    this.setState({
+      form_first_name: first_name
     });
-  }
+  };
 
-  updateLastName = (event) => {
+  updateLastName = event => {
     let last_name = event.target.value;
-    this.setState({ 
-        form_last_name: last_name 
+    this.setState({
+      form_last_name: last_name
     });
-  }
+  };
 
-  updateBio = (event) => {
+  updateBio = event => {
     let bio = event.target.value;
-    this.setState({ 
-        form_bio: bio 
+    this.setState({
+      form_bio: bio
     });
-  }
-
+  };
 
   submitUserUpdates = () => {
-      axios({
-        method: 'PATCH',
-        url: `${BASE_URL}/profile/${this.props.userid}`,
-        headers: {
-          Authorization: `Token ${localStorage.getItem("token")}` 
-        },
-        data: {
-            first_name: this.state.form_first_name,
-            last_name: this.state.form_last_name,
-            bio: this.state.form_bio
-        }
-      })
+    axios({
+      method: "PATCH",
+      url: `${BASE_URL}/profile/${this.props.userid}`,
+      headers: {
+        Authorization: `Token ${localStorage.getItem("token")}`
+      },
+      data: {
+        first_name: this.state.form_first_name,
+        last_name: this.state.form_last_name,
+        bio: this.state.form_bio
+      }
+    })
       .then(res => {
-          console.log(res)
-
-          this.setState({
-              updated_first_name: this.state.form_first_name,
-              updated_last_name: this.state.form_last_name,
-              updated_bio: this.state.form_bio
-          })
-          this.toggleEditMode();
-          // this.forceUpdate();
+        console.log(res);
+
+        this.setState({
+          updated_first_name: this.state.form_first_name,
+          updated_last_name: this.state.form_last_name,
+          updated_bio: this.state.form_bio
+        });
+        this.toggleEditMode();
+        // this.forceUpdate();
       })
-      .catch(err => console.log(err))
-
-  }
+      .catch(err => console.log(err));
```

Comment:
If there's an error, you should display that to the user, not just print it to the console